apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -22,10 +22,41 @@ const SameStateSwitchTemplate: StoryFn<typeof Switch> = () => {
     </div>
   );
 };
+const SwitchLabelIdTemplate: StoryFn<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  return (
+    <div>
+      <label htmlFor="my-switch">My switch</label>
+      <div>
+        <Switch state={state} onChange={setState} labelId="my-switch" />
+      </div>
+    </div>
+  );
+};
+const SwitchLabelWithoutNestingTemplate: StoryFn<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  return (
+    <div>
+      <label htmlFor="my-switch">
```

Comment:
The `htmlFor` shouldn't be used if there's no component with the matching `id` present. This is why this example doesn't work, browsers probably assumes it should look for an input with id "my-switch", not the one nested inside the label.

So the `htmlFor` attribute should be removed. Removing it would also reveal that what you attempted with ESLint config did not work.